### DISCUSSION TITLE
Not synced

### DISCUSSION THREAD

**Discussion by sipeneco** [Chia-Network/chia-blockchain#8247] [Support]
My Chives Blockchain is all syned (Full Node Status) the farm is farming, but my wallet isn't synced. I have a estimated Time to win of 3 day. And I am farming for 2 weeks, and my wallet has 0 XCC.

Why is taking so long to have my wallet synced?

**Comment by loppefaaret** [comment]
Ask the developers of Chives - this is Chia

**Comment by BrandtH22** [comment]
@sipeneco , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
